app/models/lookup_keys/lookup_key.rb

```diff
@@ -144,6 +144,12 @@ def overridden?(host)
     host.lookup_values.any? { |lv| lv.lookup_key_id == id }
   end
 
+  def overridden_value(host)
```

Comment:
This is basically equivalent to `overridden?`, not sure if we should implement it in terms of this instead..